### ISSUE TITLE
Activities created upon lead conversion don't get moved to selected module

### ISSUE THREAD

**Issue by EZS-JD** [salesagility/SuiteCRM#2332]
#### Issue

This is a suggestion for how lead conversion behavior would be more intuitive to me.
#### Expected Behavior

Activities (call, meeting, task, note) created during Lead conversion process get linked to the same module that is selected in the bottom-most dropdown field "Move activities to" (or copy depending on admin settings).
#### Actual Behavior

Activities (call, meeting, task, note) created during Lead conversion process get linked only to the current lead and the contact created or chosen during lead conversion process.
#### Suggestion

Either:
a) The new activities should have the "related to" field linked to chosen module set in "move activities to" or "copy activities to" field.

e.g. if I chose "Opportunity" as my module of choice to move activities to:
- new note: "related to" new Opportunity (and Contact via separate relationship field)
- new task: "related to" new Opportunity (and contact via separate relationship field)
- new meeting: "related to" new Opportunity (and lead, contact and user via separate relationships)
- new call: "related to" new Opportunity (and lead, contact and user via separate relationships)

b) Ability to choose per new activity record (call, meeting, Task, note) which record to link it to primarily (contact, account, opportunity)
#### Your Environment

- SuiteCRM Version used: SuiteCRM 7.7.4

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Activities created upon lead conversion don't get moved to selected module](https://trello.com/c/muSI5xZJ/69-activities-created-upon-lead-conversion-don-t-get-moved-to-selected-module)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
